config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  root "users#index"
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users
```

Comment:
[OPTIONAL]
You can add at this line: `` resources :users, only: [:new, :create, :edit, :update] ``

Reply:
Hello Carlos,

This is on purpose, I wanted to add a show function inside my app too. Along with `:new, :create, :edit and :update`, So there was no point to use `only:` anymore. 

Thanks,
Happy Coding!!!